akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ExternalInteractions.scala

```diff
@@ -145,7 +142,7 @@ private[akka] trait JournalInteractions[C, E, S] {
 
       if (toSequenceNr == Long.MaxValue || toSequenceNr <= lastSequenceNr) {
         setup.log.debug("Deleting events up to sequenceNr [{}]", toSequenceNr)
-        setup.journal ! JournalProtocol.DeleteMessagesTo(setup.persistenceId.id, toSequenceNr, self)
+        setup.journal.tell(JournalProtocol.DeleteMessagesTo(setup.persistenceId.id, toSequenceNr, self), self)
```

Comment:
Is the problem that when sending to classic actors there's no implicit sender in scope in typed actors? We might want to mention that in migration/interaction docs in case that's not already done.

Reply:
yes, that might be good mentioning in the coexistence page

Reply:
https://github.com/akka/akka/issues/28074